.gitmodules

```diff
@@ -490,3 +490,7 @@
 	path = apps/samson
 	url = git@github.com:zendesk/samson.git
 	branch = master
+[submodule "apps/sharetribe"]
+	path = apps/sharetribe
+	url = git@github.com:vtorves/sharetribe.git
```

Comment:
@vtorves Thanks for the contribution! Please update to use the source repo git@github.com:sharetribe/sharetribe.git